src/redisraft.h

```diff
@@ -308,6 +308,13 @@ void fsyncThreadStart(FsyncThread *th, void (*on_complete)(void *result));
 void fsyncThreadAddTask(FsyncThread *th, int fd, raft_index_t requested_index);
 void fsyncThreadWaitUntilCompleted(FsyncThread *th);
 
+typedef enum {
+    RAFT_DEBUG_MIGRATION_NONE                    = 0,
+    RAFT_DEBUG_MIGRATION_EMULATE_CONNECT_FAILED,
+    RAFT_DEBUG_MIGRATION_EMULATE_IMPORT_FAILED,
+    RAFT_DEBUG_MIGRATION_EMULATE_UNLOCK_FAILED,
```

Comment:
```suggestion
    DEBUG_MIGRATION_EMULATE_UNLOCK_FAILED,
```

We use `RAFT_` and `raft_` namespace for raft library. Probably, it's better to get rid of the prefix just to avoid confusion/collusion.